### DISCUSSION TITLE
Is there an option to automatically edit files to removed unused portions?

### DISCUSSION THREAD

**Discussion by solidcell** [peripheryapp/periphery#615] [Q&A] [answered]
I've run `scan` and now I see that there are thousands of unused entities. Is there no option or command to also update those files to remove the unused portions? Going through and doing it by hand will take days. I'm hoping I just missed this feature in the documentation.

**Comment by ileitch** [comment, marked as answer]
I'm afraid not, but I would like to implement the ability to automatically remove some of the results. Unfortunately, it'll likely never be possible to remove all results automatically, as some are too complex, i.e function parameters, not safe to assume, i.e assign-only properties.

**Reply by solidcell** [reply]
Even just the easy ones would be a huge win. As for things that aren't safe to assume, I think I would still even prefer to be able to let it make those modifications. It's easier to let it do it, try to build, and then discard any unwanted changes with Git than it is to do it manually. In any case, thanks for the tool. However with that limitation, in my case, I think the time spent deleting code manually doesn't save more time than just discovering/deleting it in the future on the go.
